packages/lesswrong/components/users/ViewSubscriptionsPage.tsx

```diff
@@ -127,6 +127,14 @@ const ViewSubscriptionsPage = ({classes}: {
   }
   
   return <SingleColumnSection>
+    <SubscriptionsList
+      title="Subscribed to Users for Susbscribed Activity Feed"
+      collectionName="Users"
+      subscriptionType="newActivityForFeed"
+      fragmentName="UsersMinimumInfo"
+      renderDocument={(user: UsersMinimumInfo) => <UsersNameDisplay user={user}/>}
+      noSubscriptionsMessage="You are not subscribed to any users' on your Subscribed Feed."
+    />
```

Comment:
Is this page shared between us and the EA forum?  If so, probably worth feature-gating